operators/src/machine_learning/onnx.rs

```diff
@@ -445,11 +438,13 @@ mod tests {
         let new_samples = Array2::from_shape_vec((rows, cols), pixels).unwrap();
 
         let outputs = session
-            .run(ort::inputs![input_name => new_samples].unwrap())
+            .run(ort::inputs! {
+                input_name => Tensor::from_array(new_samples).unwrap()
```

Comment:
ich hab das mit TensorRef gemacht. Das sollte vielleicht ne Kopie sparen. Weiß aber nicht ob das so tragisch ist. Das ist in meinem PR hier: https://github.com/geo-engine/geoengine/pull/1061
Habe da diesen PR schon rein gemerged, deshalb sollten wir das dann anschauen, wenn der hier gemerged ist.